.bsp/sbt.json

```diff
@@ -0,0 +1 @@
+{"name":"sbt","version":"1.9.3","bspVersion":"2.1.0-M1","languages":["scala"],"argv":["/Users/josephbowen/Library/Java/JavaVirtualMachines/openjdk-19.0.1/Contents/Home/bin/java","-Xms100m","-Xmx100m","-classpath","/Users/josephbowen/Library/Application Support/JetBrains/IdeaIC2023.1/plugins/Scala/launcher/sbt-launch.jar","-Dsbt.script=/opt/homebrew/bin/sbt","xsbt.boot.Boot","-bsp"]}
```

Comment:
something doesn't seem right here that you've committed sumit with your name in it